extension/src/plots/webview/messages.ts

```diff
@@ -380,6 +380,7 @@ export class WebviewMessages {
     }
 
     return {
+      errors: {},
```

Comment:
Currently, we don't send errors with paths + revs to the front end. We will need to update our plots error model to collect error information in the form of something like `{ [path: string]: { [rev: string]: string[] } }` for us to show in our error messages. 

Unfortunately, I'm off next week so I won't be able to get back to this till January 2nd.